api/Window.json

```diff
@@ -6432,7 +6432,8 @@
           ],
           "support": {
             "chrome": {
-              "version_added": "1"
+              "version_added": "1",
+              "notes": "Starting in Chrome 117, the [unload event is on the path to deprecation](https://developer.chrome.com/docs/web-platform/deprecating-unload)."
```

Comment:
We don't typically mention when a feature is deprecated in a web browser, only when its support has been removed.  I think we can remove this note.

Reply:
Updated.